src/fragments/ui-legacy/vue/overview.mdx

```diff
@@ -14,7 +14,8 @@ npm i aws-amplify-vue
 Then, alter main.js:
 
 ```javascript
-import Amplify, * as AmplifyModules from 'aws-amplify'
+import { Amplify } from 'aws-amplify'
+import * as AmplifyModules from 'aws-amplify'
 import { AmplifyPlugin } from 'aws-amplify-vue'
 import awsconfig from './aws-exports'
 Amplify.configure(awsconfig)
```

Comment:
Nit
```suggestion
import * as AmplifyModules from 'aws-amplify'
import { AmplifyPlugin } from 'aws-amplify-vue'
import awsconfig from './aws-exports'

AmplifyModules.Amplify.configure(awsconfig)
```

Reply:
Yeah I went back and forth on this one, seemed cleaner to just have two imports (to have the standard `Amplify.configure` call) but willing to change if you have strong opinions about it.

Reply:
why do we need both? what does the * as AmplifyModules accomplish here?

Reply:
So with default exports you're able to import the default and also star import the rest of the module, e.g. the original code: `import Amplify, * as AmplifyModules from 'aws-amplify'` 

This really isn't possible with named exports, so something like this is not valid for example: `import { Amplify }, * as AmplifyModules from 'aws-amplify'`. We can do what Chris suggested, but I personally thought having `AmplifyModules.Amplify.configure(awsconfig)` felt weirder than just have two imports so we can use the standard `Amplify.configure` form which is used everywhere else. Can change though if folks feel strongly about it.